requirements.txt

```diff
@@ -0,0 +1,3 @@
+boto3>=1.21.24,<2
```

Comment:
Do we need this file?
If these requirements are needed for unittesing, we should probably add them along with pytest to `requirements-dev.txt`?